samza-core/src/main/java/org/apache/samza/zk/ZkJobCoordinator.java

```diff
@@ -148,30 +131,29 @@ public void doOnProcessorChange(List<String> processors) {
   public void onNewJobModelAvailable(final String version) {
     debounceTimer.scheduleAfterDebounceTime(ScheduleAfterDebounceTime.JOB_MODEL_VERSION_CHANGE, 0, () ->
       {
-        log.info("pid=" + processorId + "new JobModel available");
+        LOGGER.info("pid=" + processorId + "new JobModel available");
         // stop current work
         if (coordinatorListener != null) {
           coordinatorListener.onJobModelExpired();
         }
-        log.info("pid=" + processorId + "new JobModel available.Container stopped.");
+        LOGGER.info("pid=" + processorId + "new JobModel available.Container stopped.");
         // get the new job model
         newJobModel = zkUtils.getJobModel(version);
 
-        log.info("pid=" + processorId + ": new JobModel available. ver=" + version + "; jm = " + newJobModel);
+        LOGGER.info("pid=" + processorId + ": new JobModel available. ver=" + version + "; jm = " + newJobModel);
 
         // update ZK and wait for all the processors to get this new version
-        ZkBarrierForVersionUpgrade barrier = (ZkBarrierForVersionUpgrade) coordinationUtils.getBarrier(
-            JOB_MODEL_UPGRADE_BARRIER);
+        ZkBarrierForVersionUpgrade barrier =
```

Comment:
Just wondering, if this is auto-format change or it is your formatting preferences?